packages/fast-components-react-base/src/listbox/examples.data.tsx

```diff
@@ -0,0 +1,82 @@
+import * as React from "react";
+import { ComponentFactoryExample } from "@microsoft/fast-development-site-react";
+import schema from "./listbox.schema.json";
+import Listbox, { ListboxManagedClasses, ListboxProps } from "./listbox";
+import { ListboxItemProps } from "../listbox-item";
+import Documentation from "./.tmp/documentation";
+
+function listboxItemPropFactory(id: string): ListboxItemProps {
+    return {
+        managedClasses: {
+            listboxItem: "listbox-item",
+        },
+        id,
+        value: id,
+        role: "option",
+        displayString: "Option-" + id,
+    };
+}
+
+const managedClasses: ListboxManagedClasses = {
+    managedClasses: {
+        listbox: "listbox",
+    },
+};
+
+const examples: ComponentFactoryExample<ListboxProps> = {
+    name: "Listbox",
+    component: Listbox,
+    schema: schema as any,
+    documentation: <Documentation />,
+    detailData: {
+        ...managedClasses,
+        typeAheadPropertyKey: "displayString",
+        children: [
+            {
+                id: "listbox-item",
+                props: {
+                    ...listboxItemPropFactory("a"),
+                },
+            },
+            {
+                id: "listbox-item",
+                props: {
+                    ...listboxItemPropFactory("b"),
+                },
+            },
+            {
+                id: "listbox-item",
+                props: {
+                    ...listboxItemPropFactory("c"),
+                },
+            },
+        ],
+    },
+    data: [
```

Comment:
I'd like to see an example where you are using something other than "displayString" as a typeahead property so we can validate that this works

Reply:
Added such a test with "value" instead